support-frontend/app/controllers/Application.scala

```diff
@@ -131,7 +131,7 @@ class Application(
 
     views.html.contributions(
       title = "Support the Guardian | Make a Contribution",
-      id = s"contributions-landing-page-$countryCode",
```

Comment:
I assume this isn't being used anywhere in the scss because it's an ID, but wanted to raise it as something to double-check

Reply:
I've searched everywhere for `contributions-landing-page`, the only place it's used is to set the id here and in the JS. Maybe we should just have a `react-root` id everywhere and not bother parameterising it